<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popup_Thu</class>
 <widget class="QDialog" name="popup_Thu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frametest">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frametest{
	background-color: #EDE4E0;
	border-radius: 15;
	border: 2px solid white;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>48</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 10px;
    min-width: 6em;
	padding-left: 10;
}

QComboBox:on {
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}

QComboBox QAbstractItemView {
    border-bottom-right-radius: 10px;
    border-bottom-left-radius: 10px;
    background: transparent;
    border: 1px solid gray;
    padding: 4px 4px 4px 4px
}

QComboBox::drop-down {
	image: url(&quot;:/Icon/Image/icons8-down-25.png&quot;);
	width:20;
	padding-left: 5
}
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <item>
     <property name="text">
      <string>Lương tháng</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Khác</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>118</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10;
background-color: white;
border: 1px solid gray;
padding-left: 5</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>48</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFile.qrc">:/Icon/Image/icons8-goal-40.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>83</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFile.qrc">:/Icon/Image/icons8-money-40.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>118</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFile.qrc">:/Icon/Image/icons8-note-40.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>83</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VNĐ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SetThu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>151</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_SetThu
{
	border: 0px solid white;
	background-color: white;
	border-radius: 10px;
}
#pushButton_SetThu::pressed
{
	background-color: #99FFFF;
	border-style: inset;
}
#pushButton_SetThu::hover
{
	background-color: #BBBBBB;
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
    <property name="icon">
     <iconset resource="ResourceFile.qrc">
      <normaloff>:/Icon/Image/icons8-next-50.png</normaloff>:/Icon/Image/icons8-next-50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_3">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>83</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10;
background-color: white;
border: 1px solid gray;
padding-left: 8</string>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_selected_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>13</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_date_selected{
	background-color: white;
	border-radius: 10;
	border: 1px solid white;
	color: black
}</string>
    </property>
    <property name="text">
     <string>09/12/2023</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_7">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>13</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10;
background-color: white;
border: 1px solid gray;
padding-left: 5</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>13</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFile.qrc">:/Icon/Image/icons8-calendar-48.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Selectday_Thu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>15</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_Selectday_Thu
{
	border: 0px solid white;
	background-color: white;
	border-radius: 10px;
}
#pushButton_Selectday_Thu::pressed
{
	background-color: #99FFFF;
	border-style: inset;
}
#pushButton_Selectday_Thu::hover
{
	background-color: #BBBBBB;
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ResourceFile.qrc">
      <normaloff>:/Icon/Image/icons8-calendar-48.png</normaloff>:/Icon/Image/icons8-calendar-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_selected_Thu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>15</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_date_selected_Thu{
	background-color: white;
	border-radius: 10;
	border: 1px solid white;
	color: black
}</string>
    </property>
    <property name="text">
     <string>09/12/2023</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="PopupselectDay_Thu">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PopupselectDay_Thu{
	background-color: white;
	border-radius: 15;
	border: 2px solid white;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>37</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFile.qrc">:/Icon/Image/icons8-calendar-48.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SelectedDay_Thu">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_SelectedDay_Thu
{
	border: 0px solid white;
	background-color: white;
	border-radius: 10px;
}
#pushButton_SelectedDay_Thu::pressed
{
	background-color: #99FFFF;
	border-style: inset;
}
#pushButton_SelectedDay_Thu::hover
{
	background-color: #BBBBBB;
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
    <property name="icon">
     <iconset resource="ResourceFile.qrc">
      <normaloff>:/Icon/Image/icons8-next-50.png</normaloff>:/Icon/Image/icons8-next-50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back_day_Thu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_back_day_Thu
{
	background-color: white ;
	border-radius: 15;
}
#pushButton_back_day_Thu:pressed
{
	background-color: white;
	border-style: inset;
}
#pushButton_back_day_Thu:hover
{
	background-color: #CCFFFF;
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ResourceFile.qrc">
      <normaloff>:/Icon/Image/back.png</normaloff>:/Icon/Image/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next_day_Thu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_next_day_Thu
{
	background-color :white ;
	border-radius: 15;
}
#pushButton_next_day_Thu:pressed
{
	background-color: white;
	border-style: inset;
}
#pushButton_next_day_Thu:hover
{
	background-color: #CCFFFF;
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ResourceFile.qrc">
      <normaloff>:/Icon/Image/next.png</normaloff>:/Icon/Image/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_select_Thu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Samsung Sharp Sans</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_date_select_Thu{
	background-color: white;
	border-radius: 10;
	border: 1px solid gray;
	color: black
}</string>
    </property>
    <property name="text">
     <string>09/12/2023</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ResourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
